src/content/en/2019/css.md

```diff
@@ -0,0 +1,115 @@
+---
+part_number: I
+chapter_number: 2
+title: CSS
+authors: [una, argyleink]
+reviewers: [meyerweb, huijing]
+---
+
+## Introduction
+Cascading Style Sheets are used to paint, format, and layout web pages. Their capabilities span concepts as simple as text color to 3D perspective. It also has hooks to empower developers to handle varying screen sizes, viewing contexts, and printing. CSS helps developers wrangle the content and ensure it's adapting properly the requesting user.
```

Comment:
"ensure it's adapting properly the requesting user."
Not really sure what this sentence is trying to express.

Reply:
fair, looks like a typo. expect a fix later. this PR is in extreme amounts of flux!

Reply:
We'll also be doing a thorough copyediting pass (#218) over all of the chapters, so it's ok to focus on a more technical review this time around. (are the statements factually correct? does the interpretation make sense? etc)

Reply:
Ok, noted.